src/Libraries/RevitNodes/Schedules/ScheduleFilter.cs

```diff
@@ -0,0 +1,161 @@
+﻿using System;
+using Autodesk.DesignScript.Runtime;
+
+namespace Revit.Schedules
```

Comment:
You overrode that namespace in Customization file last time so it falls into ScheduleView: 
![image](https://cloud.githubusercontent.com/assets/529781/23108408/ea88bffe-f6da-11e6-9814-8cfb9c9ce3f2.png)